global/scripts/FieldMapPreprocessingAll.sh

```diff
@@ -191,37 +251,26 @@ case $DistortionCorrection in
         # -- Philips Gradient Echo Field Maps --
         # --------------------------------------
 
-            ${FSLDIR}/bin/fslmaths ${MagnitudeInputName} -Tmean ${WD}/Magnitude
-            # Brain extract the magnitude image
-            ${FSLDIR}/bin/bet ${WD}/Magnitude ${WD}/Magnitude_brain -f 0.35 -m
-            ${FSLDIR}/bin/fslmaths ${WD}/Magnitude_brain -ero ${WD}/Magnitude_brain_ero
-            rm ${WD}/Magnitude_brain.nii.gz
-            mv ${WD}/Magnitude_brain_ero.nii.gz ${WD}/Magnitude_brain.nii.gz
-
-            # Take the absolute value of the magnitude data (some images used negative values for the coding)
-            ${FSLDIR}/bin/fslmaths ${WD}/Magnitude_brain.nii.gz -abs ${WD}/Magnitude_brain.nii.gz
```

Comment:
This looks like other edits that were made while you were editing this outside the repo folder.  Change all of this to what the red lines show, and remove the green lines, until this entire section disappears from the "Files Changed" tab.  This suggests that I rebased your branch while you had in-progress edits outside your clone of the repo folder.  Making your edits inside the clone folder makes it easier to avoid causing these types of issues.